packages/gatsby-theme-store/src/sdk/orderForm/useToast.ts

```diff
@@ -0,0 +1,10 @@
+// TODO remove this fake useToast
+
+export const useToast = () => {
```

Comment:
this does not should be removed?

Reply:
Nope, this function is required, for now, that we don't have a toast feature we need fake it.